kola/cluster/torcx_manifest.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type TorcxManifest struct {
+	Packages []TorcxPackage
```

Comment:
I think it reads clearer as-is even if there's duplication of a couple lines.

Reply:
I was thinking it'd be cleaner to have the machine do the duplication, rather than having a comment explaining that the fields are hand-duplicated.  But YMMV.